src/api/APIFactory.js

```diff
@@ -5,7 +5,7 @@
  */
 
 import Cache from '../util/Cache';
-import ChunkedUploadAPI from './ChunkedUpload';
+import ChunkedUploadAPI from './uploads/MultiputUpload';
```

Comment:
Can we keep the naming consistent and use ChunkedUpload instead of MultiputUpload?  This applies to all files

Reply:
yeah I'll do the renaming in another PR